docs/explanations/principles.md

```diff
@@ -0,0 +1,106 @@
+
+
+# Gutenberg Principles
```

Comment:
I think "Guiding Principles" would be better.

Reply:
See 1a648ba.